src/inactive-regions.ts

```diff
@@ -45,18 +45,18 @@ export class InactiveRegionsFeature implements vscodelc.StaticFeature {
       };
     }
   }
-  initialize(capabilities: vscodelc.ServerCapabilities,
+  async initialize(capabilities: vscodelc.ServerCapabilities,
```

Comment:
It's not clear to make whether we can make `initialize` async.

The framework is going to [call it](https://github.com/microsoft/vscode-languageserver-node/blob/d90a87f9557a0df9142cfb33e251cfa6fe27d970/client/src/common/client.ts#L2101) without using `await`... but on the other hand, I don't think it expects any particular side effect to have occurred straight away (unlike, say, `fillClientCapabilities`).

@dbaeumer, could you give us advice on this: will anything bad happen if we make a `StaticFeature` implementation's `initialize` method async?

Reply:
Had a cursory look, as you mentioned I don't think this is expected to have any immediate side-effects. Should be okay.